matrix-sdk-android/src/test/java/org/matrix/android/sdk/internal/session/room/send/LocalEchoEventFactoryTests.kt

```diff
@@ -226,16 +224,14 @@ class LocalEchoEventFactoryTests {
             ).toMessageTextContent().toContent()
         }
         return TimelineEvent(
-                root = A_START_EVENT,
+                root = A_START_EVENT.copy(
+                        type = EventType.MESSAGE,
+                        content = textContent
+                ),
                 localId = 1234,
                 eventId = AN_EVENT_ID,
                 displayIndex = 0,
                 senderInfo = SenderInfo(A_USER_ID_1, A_USER_ID_1, true, null),
-                annotations = if (textContent != null) {
```

Comment:
The remove of these lines means the `editSummary` was not used inside the related tests, am I right?

Reply:
I don't know why it was used at all like that